### PR TITLE
RESETPASSWORD: Navigate to security key after click extra security key option

### PR DESCRIPTION
**Pull request by eunjuhuss**
#### Description:
This PR is rendering the security key page if the user chooses security key option, functionality not added yet.

#### Summary:
- Added new component <SecurityKey />
- Added <ResetPasswordLayout /> for reusability within reset password layout
- Added new text for `mfa.reset-password-tapit`: 
  - Eng: Use your security key to reset password. If it has a button, tap it.
  - Swe: Använd din säkerhetsnyckel för att återställa lösenordet. Om den har en knapp, tryck på den nu.

----

Screenshot 2021-06-24 at 15 01 23

---

#### For reviewer:
- [ ] Read the above description
- [ ] Reviewed the code changes
- [ ] Navigate to the branch (pulled the latest version)
- [ ] Run the code and been able to execute the expected function
- [ ] Check any styling on both desktop and mobile sizes

### CODE REVIEW

**Review by njons — COMMENTED** [review]
Page renders as expected, but I think we have an opportunity to be clever about overlapping code/naming here.

File: src/login/components/LoginApp/ResetPassword/SecurityKey.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <ResetPasswordLayout
+      heading={props.translate("resetpw.extra-security_heading")} 
+      description={props.translate("resetpw.extra-security_description")} 
+      linkInfoText={props.translate("resetpw.without_extra_security")}
+      linkText={props.translate("resetpw.continue_reset_password")}
+    > 
+      <p>{props.translate("mfa.reset-password-tapit")}</p>
+      <div className="key-animation"  />
+      <div>
+        <form method="POST" action="#" id="form" className="form-inline">
```
Comment (njons) [inline src/login/components/LoginApp/ResetPassword/SecurityKey.js:17]: There is a component in the login flow called `SubmitSamlResponse` that might be useful to you here.

**Review by njons — COMMENTED** [review]
We have a component name clash so we might need to do some renaming.

**Comment by eunjuhuss** [comment]
I renamed `securityKey` to `ExtraSecurityToken`. We can use a component for both of login, reset password.
However at this time, I would like to keep them as separate component and look in to this later.

File: src/login/components/LoginApp/ResetPassword/SecurityKey.js (hunk 1 above)
Comment (eunjuhuss) [reply src/login/components/LoginApp/ResetPassword/SecurityKey.js]: ok, thanks
I will have a look

File: src/login/components/LoginApp/ResetPassword/ExtraSecurity.js [hunk 2]
```diff
@@ -57,26 +60,32 @@ function ExtraSecurity(props){
     }else history.push(`/reset-password/`)
   },[extraSecurity]);
 
+  const ShowSecurityKey = (e) => {
```
Comment (njons) [inline src/login/components/LoginApp/ResetPassword/ExtraSecurity.js:63]: Johan shared a link about making security keys work that might be relevant here. I'll find it and tag you on slack.

**Pull request merged**